src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -242,6 +252,46 @@ components:
       required:
         - directory
         - fileName
+
+    ProcessPrintResponseMessage:
+      description: The SQS message for processing the status change for an individual Print Request
+      type: object
+      properties:
+        requestId:
+          type: string
+          description: |-
+            The unique identifier of the Print Request
+            A 24 character hex string
+          example: 14ab70e8bd3b400fbd95246c
+        timestamp:
+          type: string
+          format: date-time
+          description: The date time the Print Provider issued this status change for the Print Request.
+          example: '2022-06-01T14:23:03.000Z'
+        statusStep:
+          type: string
+          description: The current print step this update relates to
+          enum:
+            - PROCESSED
+            - IN-PRODUCTION
+            - DISPATCHED
+            - NOT-DELIVERED
+        status:
+          type: string
+          enum:
+            - SUCCESS
+            - FAILED
+          description: Indicates if the step change was successful or not.
```

Comment:
So this represents what we receive for each PrintResponse:

```
      {
        "requestId": "507f1f77bcf86cd799439011",
        "timestamp": "2022-09-29T11:03:20.123Z",
        "statusStep": "PROCESSED",
        "status": "SUCCESS"
      }
```

Reply:
Yes, exactly. With an optional `message` field in the case of `status` being `FAILED`